Hazel/src/Hazel/Scene/SceneSerializer.cpp

```diff
@@ -181,7 +181,11 @@ namespace Hazel {
 
 	bool SceneSerializer::Deserialize(const std::string& filepath)
 	{
-		YAML::Node data = YAML::LoadFile(filepath);
+		YAML::Node data;
+		try { data = YAML::LoadFile(filepath); }
+		catch (const YAML::ParserException& ex)
+		{ HZ_CORE_ERROR("Failed to load .hazel file '{0}'\n     {1}", filepath, ex.what()); }
+
```

Comment:
I agree with [the comment](https://github.com/TheCherno/Hazel/pull/366/files#r532019974) of @PtiLuky. I did comment separately to do some formatting suggestion as well.

```suggestion
		YAML::Node data;
		try
		{
			data = YAML::LoadFile(filepath);
		}
		catch (const YAML::ParserException& ex)
		{
			HZ_CORE_ERROR("Failed to deserialize scene '{0}'\n     {1}", filepath, ex.what());
			return false;
		}
```